src/app/components/RequiredSteps.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+import { Grid } from '@mui/material';
+import { usePathname } from 'next/navigation';
+import React from 'react';
+
+import { AddUserDialog } from './dialogs/AddUserDialog';
+import { useAuth } from '../context/AuthProvider';
+import { apiUser } from '../services/endpoints/user.class';
+
+export const RequiredSteps = ({ children }: { children: React.ReactNode }) => {
+  const { user, isAuthenticated } = useAuth();
+  const pathname = usePathname();
+
+  const [isCreateUserDialogOpened, setIsCreateUserDialogOpened] =
+    React.useState(false);
```

Comment:
Hello @yassinedorbozgithub, I suggest we import everything we need from react  once and for all, to keep the code aligned with other files.

```suggestion
    useState(false);
```

Reply:
Sure we will do that